core/storage/feedback/gae_models.py

```diff
@@ -45,14 +45,18 @@
 
 # Constants used for generating new ids.
 _MAX_RETRIES = 10
-_RAND_RANGE = 127 * 127
+THREAD_ID_PREFIX_MAX_LEN = 6
```

Comment:
Not sure about this -- see above. Instead have entity_type below be one of a selected set of choices.

If you want to replace the full entity type with a shorter name, then have an explicit mapping, and a backend test that verifies that each value in the mapping is less than a given maximum length.